examples/mongodb-atlas/README.md

```diff
@@ -9,7 +9,7 @@ It deploys and scales a MongoDB cluster in the cloud.
 * Kubernetes 1.20+
 * Kanister controller version 0.91.0 installed in your cluster
 * Kanctl CLI installed (https://docs.kanister.io/tooling.html#install-the-tools)
-* MongoDB Atlas account registered with organization, project and cluster
+* MongoDB Atlas account registered with organization, project and cluster (https://www.mongodb.com/docs/atlas/getting-started)
```

Comment:
1. By `MongoDB Atlas account`, I am assuming you are talking about user account that was created in MongoDB Atlas cloud? 
2. If yes, should if we registered with organization, project and cluster? What do we exactly mean by registering here? What do we do as part of registering?

Reply:
yes user account in MongoDB Atlas. We assume Organization, Project and Cluster is already created/setup by user before trying out this BP.

Reply:
this statement is under prerequisite section. isn't it self-explanatory?